lib/web/service-worker/interceptor.js

```diff
@@ -0,0 +1,13 @@
+import { BatchInterceptor } from '@mswjs/interceptors'
+import { ClientRequestInterceptor } from '@mswjs/interceptors/ClientRequest'
+import { FetchInterceptor } from '@mswjs/interceptors/fetch'
+import { XMLHttpRequestInterceptor } from '@mswjs/interceptors/XMLHttpRequest'
+
+export const interceptor = new BatchInterceptor({
+  name: 'node-service-worker',
+  interceptors: [
+    new ClientRequestInterceptor(),
+    new FetchInterceptor(),
+    new XMLHttpRequestInterceptor()
```

Comment:
## Todo

Need to remove this. There are no XHR in Node.js. We won't be tailoring for browser-like environments like JSDOM.